views/js/submitScore.js

```diff
@@ -1,35 +1,43 @@
-const scorePopUp = () => {
-  const iframeLoc = document.getElementById(
-    "kubercade_iframe").contentWindow.location.href;
-  if (!iframeLoc.endsWith("games/pacman/")) {
-    alert("You can't submit a score if you aren't playing a game!");
-    return;
-  }
+document.addEventListener('pacmanLoad', () => {
+  const iframe = document.getElementById("kubercade_iframe");
+  iframe.addEventListener("load", () => {
+    const iframeWindow = iframe.contentWindow;
+    const origGameover = iframeWindow.gameover;
+    iframeWindow.gameover = () => {
+      score = getPacmanScore()
```

Comment:
Would it be simpler to pass `iframe` here so as not to need to find the iframe again in `getPacmanScore`?

Reply:
Done 0f8475b